Assignment1_q2_Node.py

```diff
@@ -0,0 +1,16 @@
+# Node is a unit which is used in my implementation of Singly Linked List
+class Node(object):
+
+    def __init__(self, data = None, next_node = None):
+        # Node is storing some data and a pointer to the next node
+        self.data = data
```

Comment:
Since these variables are private, you can name them `__data` and `__next` so that code outside this class cannot (accidentally) use them.